certbot/bin/run_certbot.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh
+# shellcheck disable=SC2039,SC2113
```

Comment:
```
In ./certbot/bin/run_certbot.sh line 43:
function copy_certificates {
^-- SC2113: 'function' keyword is non-standard. Use 'foo()' instead of 'function foo'.
```

Reply:
I'll raise you: https://github.com/koalaman/shellcheck/wiki/SC2113 (WTF)
Why not use `bash`?

Reply:
certbot container is based on Alpine so I just kept with sh. The function keyword is actually supported in Alpine's version of sh so this warning isn't really accurate here.